release/README.md

```diff
@@ -21,11 +21,13 @@ The LSP that powers language features is [FSAutoComplete](https://github.com/fsh
 
 The library that powers project and script loading is [proj-info](https://github.com/ionide/proj-info)
 
```

Comment:
The README changes are an artifact of merging main, not actually related to my work. I can try to make them go away if desired